### DISCUSSION TITLE
Select Type Option FIeld Invoke Mobile select tool

### DISCUSSION THREAD

**Discussion by thering1975TG** [Budibase/budibase#12987] [Help] [answered]
Doubt it is possible but thought i would see.

On most web based app forms when you select a field which is a dropdown list on mobile the options are shown on the Android / IOS style select list as if it is part of the OS and takes up most of the screen, this is intuitive and displays great

On Budibase the form on mobile displays the drop down select tool as it would on a desktop i.e part of the webpage itself. This is Okay but as the select tool is at the bottom of the page it only displays a few rows  before you have to scroll, but to see a few more lines you have to scroll the main web page a tiny bit which may show lets say 5 options to pick from rather than 3. FOr now i added extra margin at bottom of the element

So question is, is there anyway to invoke the Android / IOS select list picker, or anyway to open the dropdown list in a modal style box at least so it takes up more of the screen real estate

**Comment by ConorWebb96** [comment, marked as answer]
Hey @thering1975TG,

The pickers you are talking about are only used if you use native select html. The pickers used in Budibase are fully customised dropdown selects that mimic this functionality. As this is customised it allows us to have full control over the styling etc and therefore you won't be able to use the default Andriod / IOS selects instead. (As we just don't use it due to styling limitations.)

With the above being said, I thought there was logic that if the select was at the bottom of the page, it would open the dropdown menu above the select. (I'll verify this myself, if its not the case I'll create a feature request to get this functionality added in, as this should be what happens.)

Can you tell me what browser you are using?
What version is the browser?
Also the phone might be useful when trying to replicate.

Thanks in advance!

**Reply by thering1975TG** [reply]
HI

Thanks for the response.

The select picker does appear on screen but as it is at the bottom of that STEP page it does overspill to below the viewport so you have to scroll a little up to see the full box.

It's not a major issue as on a tablet which is where this would be used there is more screen estate and i was going to see what i could do with Custom HTML to enlarge the select dropdown area to maximise the screen real estate it uses.

My initial test was on Edge for Mobile v120.0.2210.157 on a Samsung S21 Ultra

If i test in chrome Dev panel with responsive set to s20 then the select box does appear above the element

In fact i have just tested again on mobile and it may in fact be because i added the Margin at the bottom of the select element, interestingly it appears above but if i scroll down slightly due to the additional margin it moves to below and partially off screen. I will have to mess with the design i guess to get the right amount of space between the last element and next button
